lib/app_funcs.sh

```diff
@@ -60,6 +60,17 @@ function compile_app() {
   cd - > /dev/null
 }
 
+function post_compile_hook() {
+  cd $build_path
+
+  if [ -f post_compile.sh ]; then
+    output_section "Executing post_compile.sh"
+    ./post_compile.sh || exit 1
```

Comment:
@sinetris :+1: I'm now leaning on that buildpack config. That would mean freedom to have any script in the app and not add too many Heroku/build-pack specific files.